backend/apps/menu/views.py

```
@@ -0,0 +1,191 @@
+        partial = kwargs.pop('partial', False)
+        instance = self.get_object()
+        serializer = self.get_serializer(instance, data=request.data, partial=partial)
+        serializer.is_valid(raise_exception=True)
+        self.perform_update(serializer)
+        return Response(MealSerializer(instance).data)
+
+
+class SpecialMenuRequestViewSet(viewsets.ModelViewSet):
+    serializer_class = SpecialMenuRequestSerializer
+    queryset = SpecialMenuRequest.objects.all()
+    permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)